README.md

```diff
@@ -280,12 +280,16 @@ WantedBy=multi-user.target
 
 ### Railway
 
-[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/template/DjrRRX)
+[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/template/bG6D4p?referralCode=OEa9MT)
```

Comment:
Shouldn't this simply be `https://railway.app/template/bG6D4p`?

Reply:
Yes Yes, updated it just now :)